worlds/ladx/Options.py

```diff
@@ -501,6 +501,18 @@ class AdditionalWarpPoints(DefaultOffToggle):
     """
     display_name = "Additional Warp Points"
 
+
+class ForeignItemIcons(Choice):
+    """
+    Choose how to display foreign items.
+    [Guess By Name] Foreign items can look like any Link's Awakening item.
+    [Indicate Progression] Foreign items are either a Piece of Power (progression) or Guardian Acorn (non-progression).
+    """
+    display_name = "Foreign Item Icons"
+    option_guess_by_name = 0
+    option_indicate_progression = 1

```

Comment:
Minor thing, but in my opinion, options other than Toggle and DefaultOnToggle should generally explicitly include their default

*Low priority*